### DISCUSSION TITLE
Microsoft Development ecosystem through winget

### DISCUSSION THREAD

**Discussion by woutervd** [microsoft/winget-cli#2835] [General]
Winget is a great premise and I really would like to make extensive use of it for development purposes. My main problem with it right now is the lack of package availability and configurability (visual studio components, unity engine and its plugins).

It would be great to have packages available for MSBuild, GDK, GDKX (e.g. an access restricted repo) and so on.

Are there any immediate plans for these?

**Comment by Masamune3210** [comment]
The package repository is public, anyone can add any program that adheres to the requirements of the repo. If there exists packages to install these silently already, you could even get them added yourself
